api/src/main/java/openconsensus/trace/propagation/BinaryFormat.java

```diff
@@ -100,18 +101,21 @@ static BinaryFormat getNoopBinaryFormat() {
   }
 
   private static final class NoopBinaryFormat extends BinaryFormat {
+
     @Override
-    public byte[] toByteArray(SpanContext spanContext) {
+    public ByteBuffer toByteBuffer(SpanContext spanContext) {
       Utils.checkNotNull(spanContext, "spanContext");
-      return new byte[0];
+      return EMPTY_BUFFER;
     }
 
     @Override
-    public SpanContext fromByteArray(byte[] bytes) {
+    public SpanContext fromByteBuffer(ByteBuffer bytes) {
       Utils.checkNotNull(bytes, "bytes");
       return SpanContext.INVALID;
     }
 
+    private static final ByteBuffer EMPTY_BUFFER = ByteBuffer.wrap(new byte[0]);
```

Comment:
My mind wants me to see this as the first thing in this class :)

Reply:
Hah :) Done.